glossary/general/library.md

```diff
@@ -0,0 +1,3 @@
+# Library
+
+It has numerous libraries i that let us tackle complex problems with a few lines of code. That's why Python developers are amongst the highest paid in data-related fields.
```

Comment:
```suggestion
# Library

It has numerous libraries i that let us tackle complex problems with a few lines of code. That's why Python developers are amongst the highest paid in data-related fields.
```

I feel like this doesn't start properly. Also the `i` between `libraries` and `that` makes no sense.

Also, This doesn't really feel like it is talking about `Library` because the sentence starts "It has numerous libraries"

Reply:
I think you've copied the content from the insight python/python-core/meet-python/why-learn-python.md 
into this glossary.

Reply:
You are right, my bad!